relay-dynamic-config/src/defaults.rs

```diff
@@ -176,6 +176,9 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("span.main_thread")
                     .from_field("span.sentry_tags.main_thread")
                     .when(is_mobile.clone()),
+                Tag::with_key("span.app_start_type")
```

Comment:
The `span.` prefix on other fields is historic, I think we can do without.

```suggestion
                Tag::with_key("app_start_type")
```